lib/board.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'game_logic.rb'
+class Board
+attr_accessor :map
+  def initialize
+    $map = []
```

Comment:
[Optional]  Not required to complete this project  A simpler approach would be `  $map = [1..9] ` Also Making the variable  global  allows it to be accessible 
 from anywhere in your program. You would not want someone to change the turn of the game. Encapsulation should be used here.

Reply:
done :)